index.rst

```diff
@@ -31,7 +56,7 @@ A ventilation fan was installed in the AuxTel dome to aid with bringing the dome
 Second test 08-Mar-22
 =================================
 
-This test was then repeated with the dome slit open.  The plots below show the mount tracking error increasing steadily as the fan speed is increased.  The conclusion is that the mount tracking errors are caused by turbulent wind buffeting the mount.  The elevation mount tracking errors are also very similar to that seen when mount jitter is seen when observing during times of high wind speed.
+This test was then repeated with the dome slit open.  The plots below show the mount jitter increasing steadily as the fan speed is increased.  The conclusion is that the mount jitter is caused by turbulent wind buffeting the mount.  The elevation mount jitter is also very similar to that seen when observing during times of high wind speed.
 
 Fan speed at 0%
```

Comment:
I think you can make these headings into captions, which probably makes more sense?